skops/io/_audit.py

```diff
@@ -59,3 +72,284 @@ def audit_tree(tree, trusted):
         unsafe -= set(trusted)
     if unsafe:
         raise UntrustedTypesFoundException(unsafe)
+
+
+class UNINITIALIZED:
+    """Sentinel value to indicate that a value has not been initialized yet."""
+
+
+@contextmanager
+def temp_setattr(obj: Any, **kwargs: Any) -> Generator[None, None, None]:
```

Comment:
What does `Generator[None, None, None]` mean? typehints should helps us understand the code better, not to make them more confusing :D 

Also, if everything is `Any` here, then why do we have typehints? 

This is an example where I wouldn't add any typehints here.

Reply:
The meaning for `Generator` is `Generator[YieldType, SendType, ReturnType]`, which happens to be `None, None, None` here. I agree that types don't add much here, though `**kwargs: Any` is a shorthand for `dict[str, Any]`. The difference for me between a function annotated with `Any` vs no annotation is that with the latter, I don't know if it takes any or if it was just not annotated yet, so I guess not completely useless.

I would thus keep the annotations here, but if you insist, I can remove them.

Reply:
I think typehints should be useful, if they're not, they take extra space and make things less readable.